src/kontrol/prove.py

```diff
@@ -439,15 +500,23 @@ def _init_cterm(
     setup_cterm: CTerm | None = None,
     calldata: KInner | None = None,
     callvalue: KInner | None = None,
+    summary: dict | None = None,
 ) -> CTerm:
     account_cell = KEVM.account_cell(
         Foundry.address_TEST_CONTRACT(),
         intToken(0),
         program,
-        KApply('.Map'),
-        KApply('.Map'),
+        map_empty(),
+        map_empty(),
         intToken(1),
     )
+    init_account_list: list[KInner] = [
+        account_cell,  # test contract address
+        Foundry.account_CHEATCODE_ADDRESS(map_empty()),
+    ]
+    if summary is not None:
+        init_account_list.extend(summary_to_account_cells(summary))
```

Comment:
Is extending the list safe, i.e. is it possible for summary accounts to somehow interfere with the other two accounts (e.g. same address)? If so, consider doing some validation.